web/src/main/webapp/css/classic/main.css

```diff
@@ -11,7 +11,6 @@
 
 BODY {
 	font-family: "Trebuchet MS", Arial, Helvetica, sans-serif;
-	font-size: 75%;
```

Comment:
Do you mean the icons inside the table?

I googled `psi probe screenshot` and this screenshot indicates the vertical icon aligment is slightly off to the top since version 2.0.2 (at least):
http://i1-scripts.softpedia-static.com/screenshots/PSI-Probe_1.png